lib/content-services/src/lib/document-list/components/filter-menu/filter-menu.component.html

```diff
@@ -0,0 +1,16 @@
+<button mat-icon-button [matMenuTriggerFor]="filter"
+    (click)="stopEventPropagation($event)"
```

Comment:
as a policy we agreed to call the method related to the event with 'on'TypeOfEvent in this case i expect this to be OnClick or if you want to be more detailed with the method name like : onButtonMenuClick(